src/components/SearchBar.jsx

```diff
@@ -12,45 +14,56 @@ const Row = styled.div`
 const Column = styled.div`
   flex: ${(props) => props.flex || 1};
   padding: ${(props) => props.theme.layout.space100};
-  border: 1px solid ${(props) => props.theme.searchBar.border.color};
-  border-top-left-radius: ${(props) => props.borderTopLeftRadius || '0'};
-  border-bottom-left-radius: ${(props) => props.borderBottomLeftRadius || '0'};
-  border-top-right-radius: ${(props) => props.borderTopRightRadius || '0'};
+  border: ${(props) =>
+    props.border || `1px solid ${props.theme.searchBar.border.color}`};
+  border-top-left-radius: ${(props) => props.borderTopLeftRadius || "0"};
+  border-bottom-left-radius: ${(props) => props.borderBottomLeftRadius || "0"};
+  border-top-right-radius: ${(props) => props.borderTopRightRadius || "0"};
   border-bottom-right-radius: ${(props) =>
-    props.borderBottomRightRadius || '0'};
+    props.borderBottomRightRadius || "0"};
   border-left: ${(props) =>
     props.borderLeft || `1px solid ${props.theme.searchBar.border.color}`};
   border-right: ${(props) =>
     props.borderRight || `1px solid ${props.theme.searchBar.border.color}`};
   display: flex;
+  align-items: center;
 `;
 
 const SearchBar = () => {
```

Comment:
While testing this component in my local environment, I noticed that the style didn't match what's in the Figma design. 

**Desktop View (Local vs Design)**
![Screenshot 2024-07-22 at 7 10 58 PM](https://github.com/user-attachments/assets/fb35bdcf-9d90-48bb-9ec4-cc964b04d3a3)
![Screenshot 2024-07-22 at 7 20 13 PM](https://github.com/user-attachments/assets/e9338d87-618d-4314-9a6b-ca38aca45530)

**Mobile View (Local vs Designs)**
![Screenshot 2024-07-22 at 7 11 09 PM](https://github.com/user-attachments/assets/077a5a8e-21f9-4c9c-b580-58987050df3f)
![Screenshot 2024-07-22 at 7 27 06 PM](https://github.com/user-attachments/assets/dad752f3-41f5-4e4f-840f-4c1e8cd61e49)
![Screenshot 2024-07-22 at 7 27 16 PM](https://github.com/user-attachments/assets/a7babbbd-c826-4c25-93eb-62a8964840ef)

Can you please make sure that this component matches what is illustrated in the Figma design? Let me know if you have any further questions!